data/converter/expression-data-mapping-converter.js

```diff
@@ -182,7 +184,10 @@ exports.ExpressionDataMappingConverter = Converter.specialize( /** @lends Expres
 
     compiledRevertSyntax: {
         get: function () {
-            return this._compiledRevertSyntax || (this._compiledRevertSyntax = compile(this.revertSyntax));
+
+            return this._compiledRevertSyntax || (this._compiledRevertSyntax === undefined
+                ? this._compiledRevertSyntax = this.revertSyntax ? compile(this.revertSyntax) : null
+                : null);
```

Comment:
These ternaries are getting too complicated, consider using conditional flow instead.

Reply:
I'd rather not, for perf reasons, mappings should be faster